nrf_security/src/drivers/nrf_cc3xx_platform/Kconfig

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config PSA_CRYPTO_DRIVER_ALG_PRNG_CC3XX_PLATFORM
+	bool
+	default y
+	# Only Oberon PSA core supports generate random through driver interface.
```

Comment:
NIT: This can be moved to helptext for symbol visibility

Reply:
> Bigger question: Is there a better way to phrase this since this is talking about another driver altogether

It is not talking about another driver, it is talking about the Core.
PRNG drivers can only be used from a PSA core that supports them.